src/storybook/stand-alone-documentaion/accessibility/assets/index.js

```diff
@@ -0,0 +1,27 @@
+import colorContrastImage from "./color-contrast_image.png";
+import colorContrastText from "./color-contrast_text.png";
+import hierarchyOrder from "./hierarchy_order.png";
+import imageryDecorative from "./imagery_decorative.png";
+import imageryDescriptive from "./imagery_descriptive.png";
+import imageryAlt from "./imagery_alt.png";
+import navigationFocus from "./navigation_focus-visible.png";
+import navigationTabs from "./navigation_tabs.png";
+import principleClarity from "./principle_clarity.png";
```

Comment:
did you talk with evigany about replace the images with brand-fit images?

Reply:
not yet.